### DISCUSSION TITLE
Saving Free text annotations to existing pdf removes all previous annotations

### DISCUSSION THREAD

**Discussion by protaniv** [sumatrapdfreader/sumatrapdf#3910] [General]
Same as the title. Windows 10

Having trouble seeing if my settings are incorrect. It does seem like a weird bug. I just have some annotations and when I want to save, it opens a new dialog box with all the annotations but it removes all of the previous annotations. Feels like it saves the current "updated" state over the old state and just overwrites the data instead of appending it for example. The feature works nice but the saving part is messing with me. Really annoying.

Anybody had similar experiences and did you fix them?

**Comment by GitHubRulesOK** [comment]
Always test the most current pre-release as only changes can be applied to that and yes some versions may have had related behaviour.

To be altered you need to provide a known clean source file, and a full account of what steps **repeatedly** fail to function in current [pre-release.](https://www.sumatrapdfreader.org/prerelease)
